guides/common/modules/con_package-mode-and-image-mode-hosts.adoc

```diff
@@ -7,8 +7,11 @@ It classifies all other hosts as package mode hosts.
 Image mode simplifies enterprise operations by enabling development, operations, and solution providers to use the same container-native tools and techniques to manage both applications and the underlying operating system.
 
 In image mode, users can install and manage packages, but with limitations. 
+Job templates, depending on the implementation and category, will either enable the `bootc usr-overlay` or pass the `--transient` flag to ensure that packages are installed correctly.
 Any packages installed on image mode hosts are temporary and disappear after a reboot. 
 Because of this, installing packages is generally discouraged, except for debugging purposes or, in some cases, when applying critical security patches.
+If new packages are needed on an image mode host, consider building a new container image for the host with the necessary packages built-in.
```

Comment:
What would you think about this? I'm trying to remove the need for passive voice and perhaps also simplify the sentence structure a little bit.
```suggestion
If you need new packages on an image mode host, consider building a new container image that includes the necessary packages.
```